config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/studios", to: "studios#index"
+  
+  get "/movies/:id", to: "movies#show"
+  post "/movies/:id", to: "movies#update"
```

Comment:
Your HTTP verb and controller action don't quite agree here - when you `post` something, you're creating something. If you are going to direct to an `update` controller action, you should use either `patch` or `put`.